src/Caching/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+﻿<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
+
+  <PropertyGroup>
+    <CodeAnalysisRuleset Condition="'$(IsShipping)' == 'true'">$(MSBuildThisFileDirectory)AspNet.Extensions.ruleset</CodeAnalysisRuleset>
+  </PropertyGroup>
+
```

Comment:
I am new to build.props and build.targets, and not sure why adding the package reference here instead of the csproj does not work as expected, but the same condition does not seem to filter out sample and tests projects.
```
<ItemGroup Condition...>
  <Reference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" />
</ItemGroup>
```